src/Playwright/Playwright.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we could avoid all the changes in this file.

Reply:
fixed